src/charm.lua

```diff
@@ -1122,7 +1157,9 @@ if next(SMODS.find_mod "MoreFluff") then
 
     local raw_CardArea_draw = CardArea.draw
     function CardArea:draw()
-        if self.children.area_uibox and G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_Posterization' then
+        if self.children.area_uibox and G.GAME
+            and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_Posterization'
+            and not is_charm_debuffed_safe() then
```

Comment:
This will cause weird results, since it's purely visual. If I have Crystal Ball and 6 Colours, the consumable area will show up as 6/6 during the Stoic, instead of 6/3.

Reply:
Resolved in e626f1f